### PR TITLE
Add more example datasets and narratives

### LINKED ISSUE #316: Include some examples of narratives
In episode https://carpentries.github.io/lesson-development-training/narrative.html

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #316.

### CODE REVIEW

File: episodes/narrative.md [hunk 1]
```diff
@@ -45,6 +45,27 @@ You can enable learners to make connections between what they learn in your less
...
+  They now need a system that will enable them to work collaboratibely and in parallel and track each change.
+* [Building Websites With Jekyll and GitHub lesson](https://carpentries-incubator.github.io/jekyll-pages-novice/filters/index.html) 
+  uses the narrative of creating a website for a research project to teach authoring webpages with Markdown and compiling them into a website served on GitHub.
+* [FAIR Research Software lesson](https://carpentries-incubator.github.io/fair-research-software/) uses the narrative of a poorly designed software project 
+  (which analyses [NASA’s open data on spacewalks](https://data.nasa.gov/Raw-Data/Extra-vehicular-Activity-EVA-US-and-Russia/9kcy-zwvn/data_preview) undertaken by astronauts from 1965 to 2013) that over the course of the lesson gets improved in terms of 
+  its accessibility, readability, interoperability and reusability to become more FAIR
+* This lesson uses [the recipe on how to bake sourdough cinnamon rolls](https://github.com/tobyhodges/cuddly-waffle) to 
+  teach how to use The Carpentries Workbench lesson template as we will see later on.
```
Comment (tobyhodges) [inline episodes/narrative.md:61]:
```suggestion
```

I would leave this one off the list, as 1. I may delete that repo soon, and 2. I don't think the recipe example os a particularly good fit as an example for a lesson -- the idea of a 'narrative' doesn't really hold. I will try to think of an alternative.

**Comment by tobyhodges** [comment]
Thanks @anenadic, I am going to open another PR to add to this one before we merge.

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Ok, I have made one more suggested change. If you are happy with these @anenadic, I think it is ready to go.

**Comment by anenadic** [comment]
> Ok, I have made one more suggested change. If you are happy with these @anenadic, I think it is ready to go.

Happy with all the changes - hope the build runs successfully so we can merge this PR. 👍

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**